client/my-sites/domains/domain-management/dataviews/domains-dataviews.tsx

```diff
@@ -98,6 +110,7 @@ export const DomainsDataViews = ( {
 					getDomainManagementLink( item, siteSlug, true, selectedFeature )
 			  )
 			: '';
+		// test
```

Comment:
Clean up the development leftover

Reply:
Thanks for catching! Removed in this commit: https://github.com/Automattic/wp-calypso/pull/100576/commits/90766a123264816a8465b592d7cfded07be5d98e